tests/sentry/seer/anomaly_detection/test_store_data.py

```diff
@@ -163,8 +163,8 @@ def test_anomaly_detection_fetch_historical_data_eap_spans(self):
 
         result = fetch_historical_data(self.organization, snuba_query, ["count()"], self.project)
         assert result
-        assert {"time": int(self.time_1_ts), "count": 1} in result.data.get("data")
-        assert {"time": int(self.time_2_ts), "count": 1} in result.data.get("data")
+        assert {"time": int(self.time_1_ts), "count()": 1} in result.data.get("data")
+        assert {"time": int(self.time_2_ts), "count()": 1} in result.data.get("data")
```

Comment:
Is this going to break something? The response now looks different from before?

Reply:
Nah checked with @shruthilayaj this isn't currently being used so its safe to change